containerd-shim-v2/metrics_test.go

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2017 Intel Corporation
+// Copyright (c) 2018 HyperHQ Inc.
```

Comment:
2019?

Reply:
This was copy from other file. Remove them ?

Reply:
Well, @grahamwhaley is probably the expert, but since this is a new file, even if you did base it on an existing file, there is only one test and presumably you wrote that (?) so either you (or the company you work for) have copyright on it.

Reply:
correct, you only need to retain copyright headers of others if you copied their code from the other file :-) If you *only* copied the copyright header to create a new file, then you can just list your company (as the new file will only contain your own code).